src/controller/CarRaceController.js

```diff
@@ -1,24 +1,24 @@
 const InputView = require("../view/InputView");
 const OutputView = require("../view/OutputView");
 const MESSAGES = require("../constant/Constant");
-const CarRaceGame = require("./CarRaceGame");
-const CarRaceResultRandomGenerator = require("./CarRaceResultRandomGenerator");
+const CarRaceGame = require("../domain/CarRaceGame");
+const CarRaceResultRandomGenerator = require("../domain/CarRaceResultRandomGenerator");
 
-class App {
+class carRaceController {
```

Comment:
```suggestion
class CarRaceController {
```

클래스명은 파스칼케이스로 네이밍해주세요~!

Reply:
넵! 알겠습니다!